source/welcome/log-out.rst

```diff
@@ -17,4 +17,22 @@ You can log out of Mattermost from your profile picture. Select **Log Out** to l
 
       .. image:: ../images/profile-log-out-mobile.png
         :width: 300
-        :alt: Log out of Mattermost from your profile picture.
\ No newline at end of file
+        :alt: Log out of Mattermost from your profile picture.
+
+Frequently asked questions
+--------------------------
+
+What happens when I log out of Mattermost?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you log out of Mattermost, all data related to your session is removed except a record in the app database of the server you accessed and general activity state information, such as the onboarding checklist. Your user data stored within the server database is deleted when you log out. If you were to delete the Mattermost desktop and the mobile app, the most recent server URL and state data would also be deleted.
+
+When you log out, the following additional data is also deleted:
+- On MacOS, your user credentials for this server from the keychain.
```

Comment:
In particular, I don't think we explicitly interact with the keychain even from the desktop app

Reply:
This is correct, there's no keychain added for web or desktop afaik.